package.json

```diff
@@ -58,6 +58,7 @@
     "@opentelemetry/tracing": "0.24.0",
     "@trussworks/react-uswds": "5.0.0",
     "@uswds/uswds": "3.5.0",
+    "apollo-datasource-rest": "^3.7.0",
```

Comment:
#TODO

Reply:
I pinned this on my branch, but maybe you should pin it here because this PR will be merged first?